studio/src/studio.js

```diff
@@ -256,47 +259,45 @@ class MasStudio extends LitElement {
     }
 
     get fragmentEditor() {
-        return html`<sp-overlay type="manual" ?open=${this.fragment}>
-            <sp-popover id="editor">
-                <sp-dialog no-divider>
-                    ${this.fragment
-                        ? html`
-                              <merch-card-editor
-                                  .fragment=${this.fragment}
-                                  @ost-open="${this.openOfferSelectorTool}"
-                                  @refresh-fragment="${this.refreshFragment}"
-                                  @update-fragment="${this.updateFragment}"
-                              >
-                              </merch-card-editor>
-                              <p>Fragment details (not shown on the card)</p>
-                              <sp-divider size="s"></sp-divider>
-                              <sp-field-label for="fragment-title"
-                                  >Fragment Title</sp-field-label
-                              >
-                              <sp-textfield
-                                  placeholder="Enter fragment title"
-                                  id="fragment-title"
-                                  data-field="title"
-                                  value="${this.fragment.title}"
-                                  @change="${this.updateFragmentInternal}"
-                              ></sp-textfield>
-                              <sp-field-label for="fragment-description"
-                                  >Fragment Description</sp-field-label
-                              >
-                              <sp-textfield
-                                  placeholder="Enter fragment description"
-                                  id="fragment-description"
-                                  data-field="description"
-                                  multiline
-                                  value="${this.fragment.description}"
-                                  @change="${this.updateFragmentInternal}"
-                              ></sp-textfield>
-                              ${this.fragmentEditorToolbar}
-                          `
-                        : nothing}
-                </sp-dialog>
-            </sp-popover>
-        </sp-overlay>`;
+        return this.showEditorPanel
+            ? html`<div id="editor">
```

Comment:
nit: when big block of code is involved, I recommend to use an early return instead of a ternary operation for legibility.